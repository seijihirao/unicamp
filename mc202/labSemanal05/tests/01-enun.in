create 10
add 1
add 3
add 5
add 7
add 9
remove 9
print
in 9
rank 5
rank 6
rank 7
select 3
select 4
rangecount 2 6
exit
