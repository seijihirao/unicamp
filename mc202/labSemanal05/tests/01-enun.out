S = {1,3,5,7}
belongs(9) = false
rank(5) = 3
rank(6) = 3
rank(7) = 4
select(3) = 5
select(4) = 7
rangecount(2,6) = 2
